11-embeddings-reranker-classification-tensorrt/BEI-baai-bge-m3-embedding-dense/config.yaml

```diff
@@ -10,7 +10,7 @@ model_name: BEI-baai-bge-m3-embedding-dense-truss-example
 python_version: py39
 requirements: []
 resources:
-  accelerator: A100
```

Comment:
Rejected, these are autogenerated. Will pr it for you.

Reply:
https://github.com/basetenlabs/truss-examples/blob/396085fff33b32c133eb3a3c98bfd62f76c1c772/11-embeddings-reranker-classification-tensorrt/templating/generate_templates.py#L811

Reply:
Should be outdated for this file, replaced it with h100